receiver/kafkametricsreceiver/broker_scraper.go

```diff
@@ -20,7 +20,7 @@ import (
 	"time"
 
 	"github.com/Shopify/sarama"
-	"go.opentelemetry.io/collector/pdata/pcommon"
+	"go.opentelemetry.io/collector/model/pdata"
```

Comment:
This module is not available anymore. Please use `go.opentelemetry.io/collector/pdata` instead